mantidproject/mslice
# Fix incorrect formatting for python version

**Description of work:**
In one of the last PRs a formatting error for the python version in conda_build_config was introduced. This PR fixes it.

**To test:**

Code review and check that nightly deploy worked **one day after merging this PR**.
